app/models/mixins/workflow_context_source_details_mixin.rb

```diff
@@ -0,0 +1,37 @@
+module WorkflowContextSourceDetailsMixin
+  extend ActiveSupport::Concern
+
+  def workflow_context
+    remote_ws_url = MiqRegion.my_region.remote_ws_url
```

Comment:
For modules that test this, need to modify `MiqServer` setup to ensure the server is active. (otherwise `remote_ws_url == nil`)

May want to modify the way `EvmSpecHelper` sets up the `MiqServer`

Reply:
Thanks, I should have some handling of this comes up nil anyway